Applications/DataExplorer/NetCdfDialog/NetCdfConfigureDialog.cpp

```diff
@@ -56,21 +61,20 @@ void NetCdfConfigureDialog::accept()
         valueErrorBox.exec();
     }else{
         createDataObject();
-        delete _currentFile;
         this->done(QDialog::Accepted);
     }
 }
 
 // Instructions if the Cancel-Button has been pressed.
 void NetCdfConfigureDialog::reject()
 {
-    delete _currentFile;
     this->done(QDialog::Rejected);
 }
 
 void NetCdfConfigureDialog::on_comboBoxVariable_currentIndexChanged(int id)
```

Comment:
The argument `id` isn't used in the implementation.

Reply:
It's used in for the signal that is implicitely created by this method.

I've added ```Q_UNUSED(id)``` so no warning will be displayed.